CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ OPTION(USE_CUDA "Use Cuda libs" OFF)
 OPTION(USE_CUDNN "Use Cudnn libs" ON)
 OPTION(USE_OPENCV "Use opencv" OFF)
 OPTION(USE_LMDB "Use LMDB libs" OFF)
+OPTION(USE_GLOG "Use glog libs" ON)
```

Comment:
Do we need this one?
I think we agreed to remove it and use it if cmake can detect it.